## 3/10
- saveとsave!がいまいちわからない
  - saveはfalseを返し、save!は例外を発生させる。falseにして戻して結局エラーメッセージを表示させたりするのであればそれは例外を返すのと同意では？

- createの検証エラーはできたが、なぜかulが２つできてしまった。どこで繰り返された？？liじゃなくてulなった気になる
  - ulが２つできたのでその前段の問題か？でもどこで繰り返されてるんだろう
  ★ `-` を `=` と書いていたためにおかしな動きになっていた。このエラーは `=` とすると

- update時の検証エラーがコケた。これは単純にcontrollerにupdateの修正をcreateと同じように行っていないから？
  - 明日試す